config/puma.rb

```diff
@@ -0,0 +1,43 @@
+# Puma can serve each request in a thread from an internal thread pool.
+# The `threads` method setting takes two numbers: a minimum and maximum.
+# Any libraries that use thread pools should be configured to match
+# the maximum value specified for Puma. Default is set to 5 threads for minimum
+# and maximum; this matches the default thread size of Active Record.
+#
+max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
+min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
+threads min_threads_count, max_threads_count
+
+# Specifies the `worker_timeout` threshold that Puma will use to wait before
+# terminating a worker in development environments.
+#
+worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
+
+# Specifies the `port` that Puma will listen on to receive requests; default is 3000.
+#
+port ENV.fetch("PORT") { 3000 }
+
+# Specifies the `environment` that Puma will run in.
+#
+environment ENV.fetch("RAILS_ENV") { "development" }
+
+# Specifies the `pidfile` that Puma will use.
+pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
+
+# Specifies the number of `workers` to boot in clustered mode.
+# Workers are forked web server processes. If using threads and workers together
+# the concurrency of the application would be max `threads` * `workers`.
+# Workers do not work on JRuby or Windows (both of which do not support
+# processes).
+#
+workers ENV.fetch("WEB_CONCURRENCY") { 4 }
+
```

Comment:
The rails app is not starting as mentioned by the previous reviewer. 
![image](https://github.com/Bestbynature/SpendWise/assets/78251927/54ac8b83-5d2e-4b6b-922c-8087da3eafe2)

However, according to this [issue ](https://github.com/MartinStyk/quotes-recommender/issues/4)page for Puma, for the newer versions of rails, it can be resolved by commenting out the following line in `puma.rb`: 🤗

`workers ENV.fetch("WEB_CONCURRENCY") { 4 }`

Issue fixed! 🤓
![image](https://github.com/Bestbynature/SpendWise/assets/78251927/b9592e7e-26c7-4617-81f1-ac6fac9b1871)

![image](https://github.com/Bestbynature/SpendWise/assets/78251927/cdd80dd7-762f-44d1-8475-607a23b90688)

Reply:
Thank you so much @abdulhamiid for pointing the issue out. But please I am a little bit confused about why it is running on my own machine despite the fact that the line is still available on my own local machine. I would really appreciate any explanation you can help me with on that. Thanks once again @abdulhamiid.